shacl12-core/index.html

```diff
@@ -2897,9 +2897,15 @@ <h4>Handling of Recursive Shapes</h4>
 				<h3>Conformance Checking</h3>
 				<p>
 					A <a>focus node</a> <dfn data-lt="conform|conformance">conforms</dfn> to a <a>shape</a> if and only if
-					the set of result of the <a>validation</a> of the <a>focus node</a> against the <a>shape</a> does not contain any results with a severity level representing a violation and no <a>failure</a>
+					the set of result of the <a>validation</a> of the <a>focus node</a> against the <a>shape</a> does not contain any results with a severity level of the set of disallowed levels and no <a>failure</a>
 					has been reported by it.
 				</p>
+        <p>
+          The set of disallowed severity levels is defined as the values of the property
+          <code>sh:conformanceDisallows</code> in the report, or if no triple with the property
```

Comment:
"in the report" is a bit too vague, better spell out what the subject is of the triples is.

Reply:
I've changed the wording.